openmetadata-ui/src/main/resources/ui/src/components/common/searchbar/Searchbar.tsx

```diff
@@ -82,7 +84,7 @@ const Searchbar = ({
       {label !== '' && <label>{label}</label>}
       <div className="flex relative">
         <Input
-          data-testid="searchbar"
+          data-testid={searchBarDataTestId ? searchBarDataTestId : 'searchbar'}
```

Comment:
searchBarDataTestId ?? 'searchbar'

Reply:
`searchBarDataTestId ?? 'searchbar'` in this passing and empty string will assign the empty string as tested that is why I used this approach

Reply:
I'll see if I can achieve that without manual wait